### PR TITLE
fingerprint clustering  onto develop

### PR DESCRIPTION
**Pull request by ecamenen**
Hi Steffen,
I've rebased this branch on develop and PR in develop, have I finally well understood ? :) If yes, I'll do the same for the other last tool.
Yours,
Etienne C.

### CODE REVIEW

File: config/phenomenal_tools2container.yaml [hunk 1]
```diff
@@ -443,6 +443,13 @@ assignment:
   max_pod_retrials: 1
   tools_id:
    - pathway_enrichment
+- docker_repo_override: container-registry.phenomenal-h2020.eu
+  docker_owner_override: phnmnl
+  docker_image_override: fingerprintclustering
+  docker_tag_override: latest
```
Comment (djcomlab) [inline config/phenomenal_tools2container.yaml]: Please fix the container version with the relevant container tag generated from Jenkins?

**Comment by ecamenen** [comment]
Hi David!
I've changed the "latest" tag into the proper Jenkins version. For fingerprintsubnetwork container, Jenkins had created a tag with "dev_" prefix. I've checked but I can't find why.

**Comment by djcomlab** [comment]
@ecamenen Can you confirm if you have tested this in minikube?

**Comment by ecamenen** [comment]
@djcomlab yes, I confirm.

**Pull request merged**